src/PaymentGateways/Stripe/Admin/AccountManagerSettingField.php

```diff
@@ -608,7 +610,7 @@ public function getStripeConnectButtonMarkup()
                 'website_url' => get_bloginfo('url'),
                 'give_stripe_connected' => '0',
             ],
-            esc_url_raw('https://connect.givewp.com/stripe/connect.php')
+            esc_url_raw(give(ConnectClient::class)->getApiUrl('stripe/connect.php'))
```

Comment:
Same here, use dependency injection.

Reply:
Resolved in b870f753d05dfb509090e716c9810e4640404030